app/assets/javascripts/controllers/.eslintrc.json

```diff
@@ -17,7 +17,7 @@
     "angular/on-destroy": 1,
 
     // only warnings for now
-    "angular/module-getter": 1,
+    "angular/module-getter": 0,
```

Comment:
This is just a temporary change. I just wanted to hide this warning from my editor, because everything was glowing green.